packages/lesswrong/server/rateLimitUtils.ts

```diff
@@ -89,6 +90,13 @@ async function getCommentsInTimeframe(userId: string, maxTimeframe: number) {
  * On forums other than the EA Forum, the post author is always exempt on their own posts.
  */
 async function shouldIgnoreCommentRateLimit(user: DbUser, postId: string|null, context: ResolverContext): Promise<boolean> {
+  const isRateLimitExempt = await ModeratorActions.findOne({
+    userId: user._id,
+    type: EXEMPT_FROM_RATE_LIMITS,
+    endedAt: { $gt: new Date()}
+  })
+  if (isRateLimitExempt) return true
```

Comment:
I'd put this after the non-async checks, to save some pointless round trips

Reply:
Oh, very reasonable